plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -918,6 +933,16 @@ public static Map<String, String> createTableProperties(ConnectorTableMetadata t
                 .buildOrThrow();
     }
 
+    static String toAvroCompressionCodecTableProperty(AvroCompressionKind compressionCodec)
```

Comment:
isnt this simply compressionCodec.toString() (AvroCompressionKind#codecString)?

Reply:
"uncompressed" vs  "null" will be the difference, unfortunately